gen/gen.ml

```diff
@@ -974,7 +974,8 @@ let write_wrapper funcs filename =
                 pm ")(%s) { \n" (Func.go_return_type func ~fallible:true) ;
                 if is_method && not is_inplace then
                   pm "  if del { defer ts.MustDrop() }\n" ;
-                pm "  ptr := (*lib.Ctensor)(unsafe.Pointer(C.malloc(0)))\n" ;
+                pm "  var untypedPtr uintptr\n" ;
```

Comment:
Got it! Thank you for kind description.

I'm just curious about your environment, are your app is using gotch with GPU?

I'm using gotch with CPU type and I saw repeatable malloc(0) makes continuous growing memory regardless of how big tensor allocates(because tensor memory freed by AtFree in drop)

In test sample code stopped growing because of iter reached 10000 times, more iter, more memory growing.

And I'm curious about this situation does not occur if we use GPU env.

Reply:
@pjongy ,

Yes, both in CPU and CUDA are Okay for me for long run. I use `gotch`  in transformers for BERT, Whisper, Wa2vec2, Llama2, Mistral without any problems. I am going to publish those models at `transformer` package soon.  Go GC is kind of double-blade sword, and we use both Go GC and `ts.Drop()` to clean up memory. Sometimes, I see memory blow-up in a `for-loop` and have to find a way around. Perhaps you can setup an example code that show memory blow-up so that we can discuss more.

Reply:
@sugarme 

It might okay if your long running is okay.

I'll check some more cases. (like your example in for loop)

It's okay to close both issue and this pr.
Thank you!

Reply:
In addition, could you tell me your go version currunt running?

Reply:
I am spinning both version v0.7 and v0.9 (libtorch 2.1)